python/sentiment/app.py

```diff
@@ -0,0 +1,169 @@
+import traceback
+import json
+from flask import Flask, make_response, request, jsonify
+import urllib
+
+# Using VADER, NLTK's built-in, pretrained sentiment analyzer
+# https://realpython.com/python-nltk-sentiment-analysis/#using-nltks-pre-trained-sentiment-analyzer
+from nltk.sentiment import SentimentIntensityAnalyzer
+import nltk.data
+
+# Turn on debug-level logging
+import logging
+logging.basicConfig(level=logging.DEBUG)
+
+sia = SentimentIntensityAnalyzer()
+
+app = Flask(__name__)
+
+# Report we are ready to run
+
+
+@app.route('/ready', methods=["GET"])
+def ready():
+    return make_response("OK", 200)
+
+# Handle a single chunk request
+
+
+@app.route('/process', methods=['POST'])
+def process():
+    # read the analysis type
+    analysis_type = 'word,sentence,document'
+    try:
+        payload = json.loads(request.form.get('payload'))
+        if 'analyze' in payload:
+            analysis_type = payload['analyze']
+            app.logger.info("Setting analyze to '{}'".format(analysis_type))
+    except Exception as e:
+        pass
```

Comment:
error handling here?

Reply:
Didn't include because it has a default so it doesn't matter if this fails. Updated to log the problem and default that will be used (separate PR)